.github/workflows/ruby.yml

```diff
@@ -26,8 +26,9 @@ jobs:
         # | 6.0   |  6/2023?  |   3/3023?|   2.5   | 2.7 |
         # | 6.1   |           |          |   2.5   |     |
         # | 7.0   |           |          |   2.7   |     |
+        # | 7.1   |           |          |   2.7   |     |
```

Comment:
huh. this surprised me
Looks like we are running on the suggested ruby version (3.2.z), so this looks good